src/appmixer/hubspot/engagements/CreateEmail/component.json

```diff
@@ -0,0 +1,170 @@
+    "description": "Create a email with the given properties and return a copy of the object, including the ID.",
+    "private": false,
+    "auth": {
+        "service": "appmixer:hubspot"
+    },
+    "quota": {
+        "manager": "appmixer:hubspot",
+        "resources": "requests",
+        "scope": {
+            "userId": "{{userId}}"
+        }
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "contact_id": {
+                        "type": "string"
+                    },
+                    "hs_timestamp": {
+                        "type": "string"
+                    },
+                    "hubspot_owner_id": {
+                        "type": "string"
+                    },
+                    "hs_email_direction": {
+                        "type": "string"
+                    },
+                    "hs_email_html": {
+                        "type": "string"
+                    },
+                    "hs_email_status": {
+                        "type": "string"
+                    },
+                    "hs_email_subject": {
+                        "type": "string"
+                    },
+                    "hs_email_text": {
+                        "type": "string"
+                    },
+                    "hs_attachment_ids": {
+                        "type": "string"
+                    },
+                    "hs_email_headers": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "hs_timestamp"
+                ]
+            },
+            "inspector": {
+                "inputs": {
+                    "contact_id": {
```

Comment:
Is there any particular reason why we should use snake case for inputs here?

Reply:
hmm actually there is not and will fix the eslint issue above... I just used a sample json and the [schema helper](https://appmixer-mission-control.vercel.app/schema-helper) to generate the schema

Reply:
I think because that's how HubSpot API expects it.

Reply:
> I think because that's how HubSpot API expects it.

that's  right, but it could be mapped in the component implementation